packages/flutter/lib/src/services/text_input.dart

```diff
@@ -880,6 +880,11 @@ class TextInputConnection {
     TextInput._instance._requestAutofill();
   }
 
+  void updateConfig(TextInputConfiguration configuration) {
+    assert(attached);
```

Comment:
Should this be `if (!attacted) return` ?
I remember I ran into situation when user update widget when the input connection is attached to another textfield and cause a crash

Reply:
That's a good point. I just followed what the other methods are doing.

I think this should remain as is, and the code in `EditableText.didUpdateWidget` should check before calling `updateConfig`. I'll fix it.